content/en/docs/concepts/scheduling-eviction/node-pressure-eviction.md

```diff
@@ -61,23 +69,25 @@ The kubelet uses various parameters to make eviction decisions, like the followi
 ### Eviction signals {#eviction-signals}
 
 Eviction signals are the current state of a particular resource at a specific
-point in time. Kubelet uses eviction signals to make eviction decisions by
+point in time. kubelet uses eviction signals to make eviction decisions by
```

Comment:
I don't think capitalizing kubelet is incorrect here. If you'd prefer though, you could also do `The kubelet`

Reply:
Done.